fuzzers/structure_aware/baby_fuzzer_custom_input/src/input.rs

```diff
@@ -57,6 +54,16 @@ impl CustomInput {
     pub fn optional_byte_array(&self) -> Option<&[u8]> {
         self.optional_byte_array.as_deref()
     }
+
+    /// Returns a mutable reference to the number
+    pub fn num_mut(&mut self) -> ValueMutRefInput<'_, i16> {
```

Comment:
Shouldn't th is return a `&mut i16` instead?

Reply:
No, this is the old `MappedInput` thing where I need an owned value to be able to implement functions on it and still keep the lifetime guarantees. Similar to the function returning a `MutVecInput` above.

Reply:
typedef it to `MutI16Input` to look consistent?